spec/spider-gazelle/helper_spec.cr

```diff
@@ -53,6 +62,42 @@ class HelperSpecActionController < ActionController::Base
     end
   end
 
+  @[OpenAPI(
+    <<-YAML
+      summary: Destroy route
+    YAML
+  )]
+  def destroy
+    head :no_content, description: "No content available"
+  end
+
+  get "/custom_route", :custom_route do
+    redirect_to("/hello/:id", description: "custom route redirect to show")
+  end
+
+  @[OpenAPI(
+    <<-YAML
+      summary: custom route
+    YAML
+  )]
+  def custom_route
+    previous_def
+  end
+
+  get "/alt_route" do
+    redirect_to("/hello/:id", description: "alternative route redirect to show")
+  end
+
+  @[OpenAPI(
+    <<-YAML
+      summary: alternative route
+    YAML
+  )]
+  def get_alt_route
+    previous_def
+  end
+
+  @[OpenAPI(dependencies: {create})]
```

Comment:
This annotation should be removed (and actually causes compilation issues).

Reply:
I have removed them.